homeassistant/components/coronavirus/sensor.py

```diff
@@ -47,6 +47,11 @@ def available(self):
             self.country in self.coordinator.data or self.country == OPTION_WORLDWIDE
         )
 
+    @property
+    def should_poll(self):
```

Comment:
Why are we polling when using the coordinator?

Reply:
Sorry my bad :sweat_smile:

Reply:
It looks like it was before.  ?!?  was that a bug ?